vertx-web/src/test/java/io/vertx/ext/web/handler/StaticHandlerTest.java

```diff
@@ -252,6 +252,89 @@ public void testHttp2Push() throws Exception {
     latch.await();
   }
 
+  @Test
+  public void testSkipCompressionForMediaTypes() throws Exception {
+    Set<String> compressedMediaTypes = new HashSet<>();
+    compressedMediaTypes.add("image/jpeg");
+    stat.skipCompressionForMediaTypes(compressedMediaTypes);
+    router.route().handler(stat);
+
+    waitFor(4);
+    HttpServer httpServer = vertx.createHttpServer(new HttpServerOptions().setCompressionSupported(true));
```

Comment:
@krishnarb3 this server is not properly closed at the end of the test.

Actually `WebTestBase` creates a `server` instance. Instead of creating a new variable, reuse the field:

```java
server.close();
server = vertx.createHttpServer(getHttpServerOptions().setCompressionSupported(true));
```

Then `server` will be automatically closed after the test